src/instructlab/lab.py

```diff
@@ -39,6 +37,7 @@
 
 
 @click.group(cls=DYMGroup)
+@click.version_option(package_name="instructlab")
```

Comment:
I'm curious why this line was moved? Was it required?  Or just leftover from past moves?

Reply:
no specific reason, I will put back, sorry!